src/main/webapp/script.js

```diff
@@ -12,6 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ *Hides styling for search results if a search has not been made.
+ */
+function hideElements() {
```

Comment:
Naming-wise, I think you need to be more precise about what elements this would hide. I'm actually not sure what it is. Also, I'm not sure you need this if you're not going to display the search results on this page. Let me know if I'm misunderstanding.

Reply:
I'm not going to display the search results on this page but I was going to make the new function a part of an entirely different PR. Should I just add a commit to this one?

Reply:
Ah, can you help me understand what this function is needed for, then? What's the use case where the page will load with text in the search box, and why would we want to hide the page contents in that scenario?